### PR TITLE
fix(ember): patch configuration of ember-camac-ng

### PR DESCRIPTION
**Pull request by anehx**

### CODE REVIEW

File: templates/patch-ember-camac-ng.sh.j2 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+PATCHED_CONFIG=$(cat  {{ camac_php_docroot }}/public/ember/index.html \
+    | grep '<meta name="camac-ng/config/environment"' \
+    | sed -e 's|<meta name="camac-ng/config/environment" content="\(.*\)" />$|\1|' \
+    | python3 -c "import sys, urllib.parse as up; print(up.unquote(sys.stdin.read()));" \
+    | jq '.appEnv = "{{ camac_env }}"' \
+    | python3 -c "import sys, urllib.parse as up; print(up.quote(sys.stdin.read()));")
+
+sed \
+    "s|^<meta name=\"camac-ng/config/environment\".*|<meta name=\"camac-ng/config/environment\" content=\"$PATCHED_CONFIG\" />|" \
+    -i {{ camac_php_docroot }}/public/ember/index.html
```
Comment (luytena) [inline templates/patch-ember-camac-ng.sh.j2:10]: Could you shortly explain what the script does? I'm not quite sure what all the steps are used for.

Comment (anehx) [reply templates/patch-ember-camac-ng.sh.j2:10]: It reads the config from `index.html`, parses it to a JSON object, changes properties of it via `jq` and the writes it back to the `index.html` file.

**Review by luytena — APPROVED** [review]
As far as I can tell it looks good :+1:

File: templates/patch-ember-camac-ng.sh.j2 (hunk 1 above)
Comment (luytena) [reply templates/patch-ember-camac-ng.sh.j2:10]: Thanks

**Pull request merged**